data/keys.go

```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/hex"
 
 	"github.com/Sirupsen/logrus"
-	cjson "github.com/tent/canonical-json-go"
+	"github.com/jfrazelle/go/canonical/json"
```

Comment:
I'm a little concerned about naming the package "json". It will be easy to copy/paste code to another file that imports `encoding/json` and end up with different output.